src/Altinn.Platform/Altinn.Platform.Receipt/Receipt/JWTValidationCert.cer

```diff
@@ -0,0 +1,23 @@
+-----BEGIN CERTIFICATE-----
+MIID6DCCAtCgAwIBAgIQRChfIoPK2I9HhR9Gu4/HfDANBgkqhkiG9w0BAQsFADBy
+MRMwEQYKCZImiZPyLGQBGRYDY29tMRcwFQYKCZImiZPyLGQBGRYHY29udG9zbzEU
+MBIGCgmSJomT8ixkARkWBGNvcnAxFTATBgNVBAsMDFVzZXJBY2NvdW50czEVMBMG
+A1UEAwwMUGF0dGkgRnVsbGVyMB4XDTE5MDQwOTEzMTIzN1oXDTIwMDQwOTEzMzIz
+N1owcjETMBEGCgmSJomT8ixkARkWA2NvbTEXMBUGCgmSJomT8ixkARkWB2NvbnRv
+c28xFDASBgoJkiaJk/IsZAEZFgRjb3JwMRUwEwYDVQQLDAxVc2VyQWNjb3VudHMx
+FTATBgNVBAMMDFBhdHRpIEZ1bGxlcjCCASIwDQYJKoZIhvcNAQEBBQADggEPADCC
+AQoCggEBAOQBa/dJ5VnfIjEVUB3AhiTSj1Qy4yxrUNqCp/f6hC4nqFCSbRmZoTLl
+f7GQjvink3J9s5N5rA1aDn5CaxfR835CSpzo7l9zGPOjOQ2CC5dYCS5VNFmKZx+s
+dhWM9lLIqjVjwGcp2y4E/Xtt/nhVakQMWAixg2gMTh6nE+sf3AUu62LyYoWtIjrU
+/jGl5xtsz8x3JaxK1e8T54K8OL40CCI+t3dfNdpbXqaT3hBG/d95FcZ36p9jlhql
+FUVD8z5M2YXtaQ36g2oJ/cTDkcc6bWGhCYMzAahiWH4ormSzaQdSao+5voixgxe1
+0shxp9giyl8NmBUyapSfCYzVcEjw4W0CAwEAAaN6MHgwDgYDVR0PAQH/BAQDAgeA
+MBMGA1UdJQQMMAoGCCsGAQUFBwMCMDIGA1UdEQQrMCmgJwYKKwYBBAGCNxQCA6AZ
+DBdwYXR0aWZ1bGxlckBjb250b3NvLmNvbTAdBgNVHQ4EFgQUAFi7ufJl/Pcuiv+z
+U+ozc2+a4jUwDQYJKoZIhvcNAQELBQADggEBAC5AL53kr8/y1Oe62eZNas9MGQHE
+0W7PBaHQ2wfINdRD46JVbL1Qjjxc1gwgUCac/hMYGPswPhtDFDKtXZ338VlyrOe5
+0B/9S/xeIR6NdSacPg8jdGwNMS229x07+GqbwkM7Awb5Dz3MNTlcysMj6RHvO1V3
+ZAitWbgqFxRG1RufXV0KIKpyAM8JDEUExi6FawBRCjoFK/UxtiyBHnrGCf/Unhld
+HKeaDbxGAEvq8iMyxia06FiNFIjVnhqTihmyie65y2D9GLVEevL8SBKNJny/w08U
+n5tunumuH0CboznOUkuQHrVNvfW+2lKsVn86BPqCxuOEHY8hOgrIRXsPyCE=
+-----END CERTIFICATE-----
```

Comment:
Checked with security if it's ok to push this certificate?
It's probably just a public key.

Reply:
Jupp, just a public key to validate the token :)